elasticsearch/utils/es_seed_acl

```diff
@@ -39,3 +43,12 @@ if [ ${now} -ge ${MAX_WAIT_SEED} ]; then
     exit 1
 fi
 info "Seeded the searchguard ACL index"
+
+info "Disabling auto replication"
+sgadmin -dra
```

Comment:
For each of the following operations, what happens if they fail?  The user will see errors in the logs, but should they be concerned or not?

Seems like we need to direct them to find out why the `sgadmin` script failed to apply those changes, or why the `es_util` command failed, and then re-run this script?

Reply:
If we can seed the ACL's I don't see why any of these steps would fail.  Given neither of these operations is drastically important IMO, I don't know how much thought we wish to put into error control nor if we have have a reasonable understanding of what they are or how to react.

Reply:
Checking for an error and emitting a message regarding the ramifications seems pretty easy to do.  The `sgadmin` tool is already going to emit message messages which are confusing.